tests/test_writer.py

```diff
@@ -74,27 +74,43 @@ def package_path(cwd):
 
 
 def test_write_source_with_custom_functions(
-    commands, cmd_map, path_custom_functions, cwd, directory_path, package_path
+    command_map,
+    name_map,
+    directory_path,
+    cwd,
+    package_path,
+    path_custom_functions,
+    library_name_structured,
 ):
-    cmd_path = wrt.write_source(commands, cmd_map, directory_path, cwd, path_custom_functions)
-    assert cmd_path == os.path.join(package_path, wrt.GENERATED_SRC_CODE)
-    assert os.path.isfile(os.path.join(cmd_path, "acel.py"))
+    wrt.write_source(command_map, name_map, directory_path, cwd, path_custom_functions)
```

Comment:
Do you process all the MAPDLs command at each test? That sounds a bit overkill.

You could just bring from some of the mail XMLs from ``ansys-internal/mapdl-cmd-doc`` and process only those.

Reply:
An issue has been opened to make this modification https://github.com/ansys/pyconverter-xml2py/issues/296